docs/guides/stay-up-to-date/cis/cis-1-3-0/_category_.json

```diff
@@ -1,3 +0,0 @@
-{
-  "label": "Update to CIS AWS Foundations Benchmark 1.3.0"
```

Comment:
Did you want to delete these? Are they just not necessary?

Reply:
These pages were empty, because there is also an `cis-1.3.0`, but I am not really sure about what deleting this  “.,/